### ISSUE TITLE
Q: How to add Regularizer and Constraint effect correctly to a weight/bias variable or activity?

### ISSUE THREAD

**Issue by Dong148** [tensorflow/java#532]
How to add Regularizer and Constraint effect correctly to a weight/bias variable or activity?

THANKS FOR HELP

**Comment by Craigacp** [comment]
You can add a regularization term the way you usually would, put an l2 penalty on the weights being large and add that to the loss function that you backprop.
